edb/testbase/server.py

```diff
@@ -1370,6 +1370,9 @@ async def _setup_database(dbname, setup_script, conn_args, stats):
         await admin_conn.execute(
             f'CREATE DATABASE {qlquote.quote_ident(dbname)};'
         )
+    except edgedb.DuplicateDatabaseDefinitionError:
```

Comment:
Why this bit?

Reply:
I want to be able to run the tests on a *database* that was created on and old version